packages/block-library/src/paragraph/edit.native.js

```diff
@@ -99,8 +94,6 @@ class ParagraphEdit extends Component {
 			content,
 		} = attributes;
 
-		const minHeight = styles.blockText.minHeight;
```

Comment:
It's not cleat to me why `minHeight` is still used in `HeadingEdit` and `Title` - (`const minHeight = styles.blockText.minHeight;`) -  instead we've removed it here.